.travis.yml

```diff
@@ -15,14 +15,10 @@ addons:
       # Ubuntu 16+ does not install this dependency by default, so we need to install it ourselves
       - libgconf-2-4
 
-if: tag IS present OR commit_message !~ /^chore(release)/
-
 install:
   - yarn && yarn run build
 script: yarn run ci
 
-if: tag !~ @appland/.+-v\d\.\d\.\d
-
```

Comment:
Unrelated to 100 line length but ok since both were [unused](https://applanding.slack.com/archives/CQ5JFE1DY/p1663184047625769).